### DISCUSSION TITLE
Can elektronn3 load models trained on elektronn2?

### DISCUSSION THREAD

**Discussion by jmlipman** [ELEKTRONN/elektronn3#48] [Q&A] [answered]
Hi,

I wanted to know if elektronn3 can load models trained/saved with elektronn2.

**Comment by mdraw** [comment, marked as answer]
Hi, this is not straightforward because elektronn2 is based on Theano and elektronn3 is based on PyTorch. Theoretically you can export elektronn2 network weights to an equivalent PyTorch-based network by copying them by hand with NumPy but I would recommend just retraining from scratch with elektronn3 instead.
